source/dub/project.d

```diff
@@ -724,16 +741,18 @@ class Project {
 
 		void determineDependencyConfigs(in Package p, string c)
 		{
+			auto pname = package_names[p];
```

Comment:
I don't think this is superior to `p.name`.

Reply:
The profiler thinks otherwise.